scripts/gen_flatbuffer.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+echo "Running gen_flatbuffers.sh OSTYPE is $OSTYPE"
+ROOT=$(pwd)
+FF_LOCATION="$ROOT/third_party/flatbuffers"
+cd "$FF_LOCATION" || exit
+mkdir build
+cd build || exit
+py() { command python "$@"; }
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_OSX_ARCHITECTURES=x86_64 ..
```

Comment:
Please compile flatbuffers as fat binary for MacOS, also as suggested in initial review, re-use CMAKE_GENERATOR from the parent project
```suggestion
cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_OSX_ARCHITECTURES=x86_64,arm64  ..
```